functions/setup.sh

```diff
@@ -8,9 +8,23 @@ setup() {
     ${warning}Web Devkit setup wizard${end}${NL}You have to complete this setup wizard in order to use site cloner."
     
     # Adds an alias
-    bash_profile_path="/Users/$USER/.bash_profile"
-    if ! grep -qF "alias $prefix" $bash_profile_path ; then
-        printf "${NL}alias $prefix=\"bash $scriptpath/start.sh\"" >> $bash_profile_path
+    case $0 in
```

Comment:
This $0 seems to be the full path to the start.sh file. Could $SHELL be used in stead?

Reply:
@torbratsberg New commit added!